altsrc/yaml_file_loader.go

```diff
@@ -78,6 +80,11 @@ func loadDataFrom(filePath string) ([]byte, error) {
 			return nil, fmt.Errorf("Cannot read from file: '%s' because it does not exist.", filePath)
 		}
 		return ioutil.ReadFile(filePath)
+	} else if runtime.GOOS == "windows" && strings.Contains(u.String(), "\\") {
```

Comment:
Aha, interesting, do you mind leaving a comment to that effect? That isn't immediately obvious looking at this code.